data/human/post-war reactions.txt

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) 2022-2023 by an anonymous author
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+########################################################################
+
+# This file contains human reactions to the pug war, and events
+# leading up to it.  It lets us know how regular, everyday people felt
+# about the war. This leads to the reveal of the Hai, who are unable
+# to stay hidden with so many people thinking about aliens.
+
+mission "00 Post-War Reactions Begin"
+	landing
+	invisible
+	to offer
+		has "main plot completed"
+	on offer
+		event "post-war reactions begin" 20 30
+		fail
+
+event "post-war reactions begin"
+
+
+mission "War Reactions: Asteroid Miners"
+	landing
+	minor
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		# These miners were far from the war the whole time.
+		attributes "deep" "paradise" "north"
+		not attributes uninhabited
+	to offer
+		has "event: post-war reactions begin"
+		random < 30
+	on offer
+		set "pug media coverage"
```

Comment:
The variable is misnamed, and I kept it that way to avoid breaking old saves. It should be "awareness of alien activity" since it indicates the player knows that human awareness is increasing. The reason for the incorrect name, "pug media coverage," is that it is replacing a mission that just showed pug media coverage.

If you have suggestions on how to rename this without breaking old saves, I'm open to them.

Reply:
Also, these changes, plot-wise, should be entirely separate from Hai Reveal, and aren't specific to the Pug War. They show how regular people were affected by the events of the main campaign.

Reply:
HR has only ever been in continuous, and so we should not be keeping improper names for the sake of save compatibility. Part of playing on continuous is the expectation that things might break.

Reply:
Okay, I just removed the "Fix for pug media coverage" mission. There is now no fix for broken saves.